### PR TITLE
CleanMediaNet Bid Adapter : add userid support and update testing

### PR DESCRIPTION
**Pull request by CleanMediaNet**
## Type of change
- [X ] Build related changes

## Description of change
Adapter Schain and user id support has been added
MD setup updated
test file updated

### CODE REVIEW

File: modules/cleanmedianetBidAdapter.js [hunk 1]
```diff
@@ -23,53 +44,50 @@ export const helper = {
...
-      const baseEndpoint = 'https://bidder.cleanmediaads.com';
-      const rtbEndpoint =
-        `${baseEndpoint}/r/${
-          params.supplyPartnerId
-        }/bidr?rformat=open_rtb&reqformat=rtb_json&bidder=prebid` +
-        (params.query ? '&' + params.query : '');
-      let url = bidderRequest.refererInfo.page;
-
+      const {adUnitCode, auctionId, mediaTypes, params, sizes, transactionId} = bidRequest;
+      const baseEndpoint = params['rtbEndpoint'] || ENDPOINTS['cleanmedianet'];
```
Comment (Fawke) [inline modules/cleanmedianetBidAdapter.js:84]: Can you please enforce the value from `params[rtbEndpoint]` is using `https:` protocol.

**Review by Fawke — COMMENTED** [review]
Hi @CleanMediaNet,

This change looks fine. I just added a small comment to ensure that the endpoint stays https.
Apart from that, I'm trying to re-trigger the circle ci tests, but having some issues. 

Post a successful re-run of the Circle CI unit tests, I'll merge this.

**Comment by CleanMediaNet** [comment]
> Hi @CleanMediaNet,
> 
> This change looks fine. I just added a small comment to ensure that the endpoint stays https. Apart from that, I'm trying to re-trigger the circle ci tests, but having some issues.
> 
> Post a successful re-run of the Circle CI unit tests, I'll merge this.

I'm also having trouble getting Circle CI to re-run, it instantly fails.  Do you have any advice or should I do a fresh fork and re-submit?  
It all tests great locally but the Circle CI test is not running it, any advice or direction would be appreciated.
Thanks

**Comment by ChrisHuie** [comment]
> > Hi @CleanMediaNet,
> > This change looks fine. I just added a small comment to ensure that the endpoint stays https. Apart from that, I'm trying to re-trigger the circle ci tests, but having some issues.
> > Post a successful re-run of the Circle CI unit tests, I'll merge this.
> 
> I'm also having trouble getting Circle CI to re-run, it instantly fails. Do you have any advice or should I do a fresh fork and re-submit? It all tests great locally but the Circle CI test is not running it, any advice or direction would be appreciated. Thanks

Just tried kicking off myself but it didn't work which is strange. Usually this problem comes up because you are following a fork of the project in CircleCI instead of the main Prebid.js repo. You can find the projects you follow on the left and side of circleci. Unfollowing your fork should fix the issue

**Comment by CleanMediaNet** [comment]
> 

It is un-followed.

**Comment by Fawke** [comment]
@CleanMediaNet I was having another look at the PR today, and wanted to approve it, but I still see that you haven't added a check to make sure the endpoint url coming from `params[rtbEndpoint]` is always https.

I've run the tests locally, and they're all passing, so I don't think we'd need circle ci checks for this. If @ChrisHuie can somehow make it work, then it's great, otherwise, it's fine to merge post the change I suggested above.

**Comment by CleanMediaNet** [comment]
> Fawke

@Fawke, I've updated to only use the HTTPS endpoint and not allow an endpoint to be passed.  Let me know if there is anything else you see that needs updated.

**Review by Fawke — APPROVED** [review]

**Comment by CleanMediaNet** [comment]
@Fawke is there anything else we need to do to get the adapter to show up on the download page?

**Pull request merged**